templates/home/_before_after.html.twig

```diff
@@ -0,0 +1,78 @@
+
+<section id="before_after" class="before_after">
+    <!-- Titre de section -->
+    <div class="pt-2">
+        <h2 class="text-light w-100 text-center">ILS ONT <b>RÉUSSI</b> !</h2>
+    </div>
+    <!-- Carousel -->
+    <div id="carouselExampleControls" class="w-100 carousel slide" data-ride="carousel">
+        <div class="carousel-inner">
+        {% for transformation in transformations %}
+            {% if loop.index is odd %}
```

Comment:
trop de code repété. Je vois pas trop à quoi te sers le even/odd ? mais si tu en as vraiment besoin, ça ne devrais pas etre pour répéter quasi tout le code dans le if et le else. 
DRY ;-)

Reply:
Optimisation is ok, je crois...